ui/settings_page.go

```diff
@@ -123,17 +128,21 @@ func (pg *settingsPage) general() layout.Widget {
 				}),
 				layout.Rigid(pg.lineSeparator()),
 				layout.Rigid(func(gtx C) D {
-					return pg.subSection(gtx, "Currency conversion", func(gtx C) D {
-						return layout.Flex{}.Layout(gtx,
-							layout.Rigid(func(gtx C) D {
-								txt := pg.theme.Body1("None")
-								txt.Color = pg.theme.Color.Gray
-								return txt.Layout(gtx)
-							}),
-							layout.Rigid(func(gtx C) D {
-								return pg.currencyConversion.Layout(gtx)
-							}),
-						)
+					return decredmaterial.Clickable(gtx, pg.currencyConversion, func(gtx C) D {
+						return layout.Inset{Top: values.MarginPadding5, Bottom: values.MarginPaddingMinus5}.Layout(gtx, func(gtx C) D {
+							return pg.subSection(gtx, "Currency conversion", func(gtx C) D {
```

Comment:
why wasn't the clickableRow method used for currency conversion. It looks like it would work here.

Reply:
Yes it would, but the functionality hasn't been implemented yet, once the functionality is implemented, we can then pass in the required label as the parameter

Else I'd have to put an if else statement in the clickableRow function to accommodate the current conversion